ctru-sys/bindgen.sh

```diff
@@ -29,6 +29,6 @@ bindgen "$DEVKITPRO/libctru/include/3ds.h" \
     -D__3DS__ \
 > src/bindings.rs
 
-cargo run --bin docstring-to-rustdoc --features="build-binary" -- src/bindings.rs
+cargo docstring-to-rustdoc src/bindings.rs
```

Comment:
Looks good to me, but this might need to be something like 
```sh
cargo run --package ctru-sys/docstring-to-rustdoc -- src/bindings.rs
```
Unless you actually install the binary at some point

Reply:
I tested it, and because of the alias added in the ``cargo/config.toml`` it works without any problem

Reply:
Ah, I see. That file isn't checked in as part of the repository due to this `.gitignore` entry: https://github.com/rust3ds/ctru-rs/blob/master/.gitignore#L3

So I don't think it will work for anyone else unless the command is changed or the `config.toml` gets checked in (not my preference personally, since I tend to use the config file for random stuff like patches, etc.)

Reply:
Ahhhhh I didn't realize that! I'll change it then

Reply:
Done: dff7ee19c6ef04fd868e0c85585c242550260454